src/compensations/Compensations.tsx

```diff
@@ -99,7 +99,7 @@ const Compensations = ({ compensations, locations }: CompensationsProps) => {
       <Text type="h1">{compensations.basicTitle}</Text>
       <RadioButtonGroup
         id="location-group"
-        label="Choose your location"
```

Comment:
this reminds me, we need to find all hardcoded strings that are in need of translations!